src/lib/OS/Process.cc

```diff
@@ -4,6 +4,7 @@
 
```

Comment:
Not in the git diff but we've removed the returning of the UNMAP fault from the page table delete mapping function call but still detect it?

Reply:
I think its better to have the checks and they are un-used for now, than forget to add them in future